README.md

```diff
@@ -1,38 +1,42 @@
-# HTTP
+# Http
 
 _friend of the web since ‘91_
 
 ## Authors
 
-By Jeff Cross with much input from Caitlin Potter, Ben Lesh, Igor Minar, and other Angular team
-members.
+By [Jeff Cross](https://github.com/jeffbcross) with much input from
+[Caitlin Potter](https://github.com/caitp), [Ben Lesh](https://github.com/blesh),
+[Igor Minar](https://github.com/igorminar), and other Angular team members. Jeff is the one
+responsible for bad ideas and errors.
 
 ## Overview
 
 Apps need a way to retrieve and change and assets from other places. Http is one of many reasonable
 ways to accomplish this, and the most widely-available option for data access in the wild world wide
-web. This doc is concerned with the http collection of modules that will be part of Angular2,
-heretofore referred to as HTTP.
+web. This doc is concerned with the Http collection of modules that will be part of Angular2,
+heretofore referred to as Http.
 
 The primary use cases for this library are:
   * Transactional data access, as opposed to continuous access provided by other means like
-    WebSockets and Server-Sent Eventse * Facilitating fetching and manipulating of remote application models
-  * Loading media into an applicatione * Uploadine  e files to servers
+    WebSockets and Server-Sent Events
+  * Facilitating fetching and manipulating of remote application models
```

Comment:
I'll come up with something that encompasses creating/updating/patching. And the point should be made clear that the lib doesn't do much other than support standard HTTP verbs, doesn't actually support complex RPCs.